frontend/libs/studio-components/src/components/StudioToggleableTextfield/StudioTextfieldToggleView/StudioTextfieldToggleView.tsx

```diff
@@ -18,7 +18,7 @@ export const StudioTextfieldToggleView = ({
     <StudioButton className={className} onClick={onClick} {...rest}>
       <span className={classes.viewModeIconsContainer}>
         <KeyVerticalIcon data-testid='keyIcon' aria-hidden />
-        {children}
+        <div className={classes.componentIdInViewMode}> {children}</div>
```

Comment:
I'm not sure we should use this css inside the Studio component 🤔 Since the StudioComponent should be an context-independent component that would not know here it is used

Reply:
I am not sure if it affects other places. What do you think, @framitdavid?

Here, I removed the CSS from StudioTextfieldToggleView and applied it to the upper field PropertiesHeader, which includes the StudioTextfieldToggleView. However, we cannot access the text inside the fields to break it down into new line because it belongs to StudioTextfieldToggleView.  I think we can also add a validation for StudioButton to limit the charachters numbers , It will be like that when moving css to PropertiesHeader:


https://github.com/Altinn/altinn-studio/assets/90609090/7a9fcde8-1009-4cc9-b3ca-f62f2ac7be37

Reply:
Do we need to show the whole text, or can an we cut the part that doesn´t fit in the frame? Like we do in the select field :)

Reply:
@JamalAlabdullah I suggest using the CSS property `text-overflow` to support ellipsis. This way, we won't need to display the entire content if there's insufficient space. It's important to use CSS for ellipsis rather than truncating the text with JavaScript, as the available space can change dynamically. For example, once we implement the column resizing feature, users will be able to adjust the column width themselves. By using CSS for ellipsis, it will automatically detect when and where ellipsis is needed, ensuring the text is displayed appropriately.

Reply:
> Do we need to show the whole text, or can we cut the part that doesn’t fit in the frame, like we do in the select field?

Fixed as you mintioned😊

Reply:
> I'm not sure we should use this css inside the Studio component 🤔 Since the StudioComponent should be an context-independent component that would not know here it is used

Removed `className `and `div` 😊

Reply:
Should we add title prop to the component so we can hover to see the whole id? 🙈 
Unless it is not already implented? 🤔 Cant see it in dev at least

Reply:
I addedt it.